app/helpers/date_helper.rb

```diff
@@ -0,0 +1,13 @@
+module DateHelper
+
+  def days_ago_in_words(date)
+    distance_in_days = Date.today - date
```

Comment:
@boblail This returns a fraction (4/1) or (10/1). Just need to wrap it in a `.to_int`

```ruby
distance_in_days = (Date.today - date).to_i
```